neard/src/cli.rs

```diff
@@ -373,6 +412,8 @@ impl RunCmd {
             actix::System::current().stop();
         });
         sys.run().unwrap();
+        info!(target: "neard", "Waiting for RocksDB to gracefully shutdown");
+        RocksDB::block_until_all_instances_are_dropped();
```

Comment:
which threads are still 'alive' when we're at this point?

Reply:
In the experiments it was always one of `ViewClientActor` threads. RocksDB gets dropped when all references are dropped. Therefore the thread of the last dropped actor is the thread dropping RocksDB.